### ISSUE TITLE
Does not compile current version with `make`

### ISSUE THREAD

**Issue by e3krisztian** [mptre/yank#25]
Well, actually I could compile, but only after removing `-std=c99` from `Makefile`, this is the unmodified compile attempt:

```
$ make
cc -Os -pedantic -std=c99 -Wall -Werror -Wextra yank.c -o yank  -DVERSION=\"0.6.2\" -DYANKCMD=\"xsel\"
yank.c: In function ‘args’:
yank.c:96:14: error: implicit declaration of function ‘getopt’ [-Werror=implicit-function-declaration]
  while ((c = getopt(argc, (char * const *)argv, "ilvxd:g:")) != -1) {
              ^
yank.c:100:13: error: ‘optarg’ undeclared (first use in this function)
    s = ator(optarg);
             ^
yank.c:100:13: note: each undeclared identifier is reported only once for each function it appears in
yank.c:130:6: error: ‘optind’ undeclared (first use in this function)
  if (optind < argc && strncmp(argv[optind - 1], "--", 3))
      ^
cc1: all warnings being treated as errors
Makefile:10: recipe for target 'yank' failed
make: *** [yank] Error 1
```

I have tried this on an up-to-date Arch Linux, gcc 5.3.0.

**Comment by e3krisztian** [comment]
Oh, this is quite recent - caused by the last commit: https://github.com/mptre/yank/commit/3214802d5259d51c27d85a0c052171ebf87fc0c6

**Comment by mptre** [comment]
My bad, fixed by 9923188.
